pkg/apiserver/apiserver.go

```diff
@@ -129,6 +132,11 @@ func installAPIGroup(s *APIServer, c completedConfig) error {
 	cpStorage["networkpolicies"] = networkpolicy.NewREST(c.extraConfig.networkPolicyStore)
 	cpGroup.VersionedResourcesStorageMap["v1beta1"] = cpStorage
 
+	// TODO: networkingGroup is the legacy group of controlplane NetworkPolicy APIs. To allow live upgrades from up to
+	//  two minor versions, the APIs must be kept for two minor releases before it can be deleted.
```

Comment:
the whitespaces look weird here?

Reply:
It's the Goland's format for a TODO, I updated them to keep align in latest patch.